mczernek-exchange-application/kubefiles/exchange-app-template.yml

```diff
@@ -0,0 +1,255 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: financial-app
+  annotations:
+    description: "Financial sample application for Red Hat Training"
+objects:
+  - apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: frontend-gw
+    spec:
+      selector:
+        istio: ingressgateway # use istio default controller
+      servers:
+      - port:
+          number: 80
+          name: http
+          protocol: HTTP
+        hosts:
+        - "*"
+  - apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: frontend-vservice
+    spec:
+      hosts:
+      - "*"
+      gateways:
+      - frontend-gw
+      http:
+      - match:
+        - uri:
+            prefix: /frontend
+        rewrite:
+          uri: /frontend
```

Comment:
I've just tested that this is not necessary. I assumed that the prefix will be lost after re-routing, but it is kept. So the rewrite and uri lines can be deleted.

Reply:
Thanks Marek. Fixed!